front/lib/api/assistant/participants.ts

```diff
@@ -69,7 +69,6 @@ export async function fetchConversationParticipants(
     where: {
       conversationId: conversation.id,
     },
-    order: [["rank", "ASC"]],
```

Comment:
Why change that?

Reply:
At @flvndvd request, and I agree with him, to speed up the call. The ordering is not needed and prevents the index from working efficiently (admittedly out of scope of PR but well)